### PR TITLE
Foundation launch changes

### LINKED ISSUE #128: Updates for Foundation Announce
Opening this issue for tracking. At Todd Benzies request, I am currently working to make a number of short-term updates to the website for the Foundation announce on 2015-06-16. These include:
1. Removing the "Sponsored by Joyent" branding on the various pages
2. Adding the usual Linux Foundation boilerplate header/footer
3. Adding the info@nodejs.org and trademark@nodejs.org contact points
4. Updating the Trademark Guidelines to reflect the Foundation's license to use the Node.js trademark
5. Adding an information request form for Foundation membership.
6. Updating the initial list of Foundation members.

These changes will need to go live on Tuesday, June 16th. However, as I'm going to be on vacation that day, I'll need someone else to actually review and land the changes. This is a time sensitive update and the actual list of Foundation members is confidential until the announce actually happens so the PR cannot be posted and landed until June 16th. I'll will open the PR Tuesday morning immediately before I head out the door. 

/cc @joyent/node-collaborators @mikeal @misterdjules @rvagg @Fishrock123

### PR DESCRIPTION
**Pull request by rvagg**
Closes #128

I've just pushed the attached commits live to nodejs.org as per instructions from @jasnell and @misterdjules. I've made some very minor tweaks & fixes in e23bae8 on top of @jasnell's original.

Can someone with access please merge this asap?

### CODE REVIEW

**Comment by rvagg** [comment]
added 7f222c2 which removes the details about board members at request of Todd Benzies as these are not current

**Comment by fhemberger** [comment]
@rvagg LGTM, some remarks:
- Do you know if the mentioned email addresses (trademark/info/membership @ nodejs.org) are already operational?
- Removed nonexistent background image "di_dark_gray.png" from foundation footer CSS (just some diagonal stripes).
- Adjust footer max-width to match the rest of the site
- Fixed typo "Poliicies"
- contribute/index.md: Added issue link for the current nodejs.org website (this repo).
- Fixed makefile, so the foundation member logos end up in the correct directory (before they ended up in `/images/logos/logos`)

File: doc/download/index.html [hunk 1]
```diff
@@ -131,6 +141,16 @@
       Note: Python 2.6 or 2.7 is required to build from source tarballs.
     </p>
 
+    <h2 style="margin-top:1em" id="iojs">io.js</h2>
+    <p>
+      In May 2015, the <a href="http://iojs.org">io.js</a> project TSC
+      voted to join the Node.js Foundation and merge back with Node.js.
+      While the project streams are being
+      <a href="http://github.com/node.js/node">converged</a>,
```
Comment (evanlucas) [inline doc/download/index.html:149]: Shouldn't the link be to `https://github.com/nodejs/node`?

Comment (fhemberger) [inline doc/download/index.html:149]: Good catch, fixed it.

**Comment by fhemberger** [comment]
Merged, still needs to be deployed, pinging @misterdjules.

**Comment by rvagg** [comment]
I have access to deploy and have also pushed your fix to the "converged" repo link

**Comment by fhemberger** [comment]
Great, thanks.

**Comment by rvagg** [comment]
@fhemberger if you're around, could you help with changing the footer to say `&copy; 2015 Node.js Foundation.` instead of `&copy; 2015 Linux Foundation.`. The LF folks picked that up and requested the change via email -- I just don't have a handle on the templating in this repo. Ping me in here when it's on `master` and I'll do a deploy if I'm still awake (heading to bed in a moment).

**Comment by misterdjules** [comment]
@rvagg @fhemberger @joyent/node-website Done with https://github.com/joyent/node-website/commit/a532ff4039ca8ced204dd7a5b258f50ae3231021.

**Comment by misterdjules** [comment]
And deployed too.

**Comment by fhemberger** [comment]
Cool, thanks @misterdjules!

**Comment by Krinkle** [comment]
Most (if not all) existing banners on the site default to or have high pixel density versions. Most logos and images added in this commit also look pretty slick and HiDPI-friendly. However, the Linux Foundation banner added atop every page is not and looks a bit cheap in comparison now. Do we have a better rendering?

screen shot

**Comment by snostorm** [comment]
@Krinkle funny you should ask, I just was trying to find a better source logo myself about 20 minutes ago. (I'm finally digging in deep in to the state of this repo for the first since the merger.)

Let's open a new issue for this one. I'll open it with more details there and x-link.

**Comment by snostorm** [comment]
Correction @Krinkle, an issue already exists at https://github.com/joyent/node-website/issues/133 -- Somebody also already referenced one of the images I tracked down ;)  We can continue there.

**Pull request merged**